admin/user.crt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
While checking certificates in is fine, and necessary, we should also make sure to include the script for reproducing them.